[IMP] essentials: iap graphics update and rewrite (#8812)
odoo/documentation: content/applications/essentials/in_app_purchase.rst

<https://iap.odoo.com/iap/all-in-app-services>`_.

Use IAP services
----------------

|IAP| services are automatically integrated with Odoo, and do **not** require users to configure any
settings. To use a service, simply interact with it wherever it appears in the database.

.. example::
   The following flow focuses on the *SMS* |IAP| service being used from a contact's record.

   This can be done by clicking the :guilabel:`📱 (phone) SMS` icon within the database.

   .. image:: in_app_purchase/sms-icon.png
      :align: center
      :alt: The SMS icon on a typical contact information form located within an Odoo database.

   One way to utilize the *SMS* |IAP| service with Odoo is showcased in the following steps:

   First, navigate to the :menuselection:`Contacts application`, and click on a contact with a
   mobile phone number entered in either the :guilabel:`Phone` or :guilabel:`Mobile` field of the
   contact form.

   Next, hover the mouse over the :guilabel:`Phone` or :guilabel:`Mobile` field, and a :guilabel:`📱
   (phone) SMS` icon appears to the right.

   Following this, click the :guilabel:`📱 (phone) SMS` icon, and a :guilabel:`Send SMS Text

>    Message` pop-up window appears.

Reviewer: I tried several diff. 15 runbots but the windows all just say "Odoo" at the top- not sure if that's runbot or not, but just want to make sure the pop-up window is named correctly. 
![image](https://github.com/odoo/documentation/assets/121518652/fd383b71-0b42-4de8-8daa-79b2086a62b6)

Credits are purchased in *Packs* from the `Odoo IAP Catalog
<https://iap.odoo.com/iap/all-in-app-services>`_, and pricing is specific to each service.

.. example::
   The `SMS service <https://iap.odoo.com/iap/in-app-services/1>`_ has four packs available, in
   denominations of:

   - :guilabel:`Starter Pack`: 10 credits
   - :guilabel:`Standard Pack`: 100 credits
   - :guilabel:`Advanced Pack`: 500 credits
   - :guilabel:`Expert Pack`: 1,000 credits

   .. image:: in_app_purchase/packs.png
      :align: center
      :alt: Four different packs of credits for the SMS IAP service.

   The number of credits consumed depends on the length of the SMS and the country of destination.

   For more information, refer to the :doc:`SMS Pricing and FAQ
   </applications/marketing/sms_marketing/pricing/pricing_and_faq>` documentation.

.. _iap/buying_credits:

Buy credits
-----------

If there are not enough credits to perform a task, the database automatically prompts the purchase
of more credits.

Users can check the current balance of credits for each service, and manually purchase more credits,

> by navigating to the :menuselection:`Settings app --> Contact --> Odoo IAP --> View My Services`.

Reviewer: ```suggestion
by navigating to the :menuselection:`Settings app --> Contacts --> Odoo IAP --> View My Services`.
```

Doing so reveals an :guilabel:`IAP Account` page, listing the various |IAP| services in the

> database. From here, click an |IAP| service to open its :guilabel:`Account Information` page, where

Reviewer: In 15 it looks like it's either "Odoo IAP" or "My Services" - but I don't have any credits to see this in these runbots, so I can't say what it looks lif eif ther are credits purchased. 

![image](https://github.com/odoo/documentation/assets/121518652/83a63195-3567-4f67-9985-247735224950)

additional credits can be purchased.

Manually buy credits
~~~~~~~~~~~~~~~~~~~~

To manually buy credits in Odoo, follow these steps:

First, go to the :menuselection:`Settings application` and type `IAP` in the search bar. Under the

> Contacts section, where it says :guilabel:`Odoo IAP`, click :guilabel:`View My Services`.

Reviewer: I like that these are different steps- I just worry about cinfusing the rader. Maybe state that they cna either search or scroll down to the section?

      :align: center
      :alt: The Settings app showing the Odoo IAP heading and View My Services button.

Doing so reveals an :guilabel:`IAP Account` page, listing the various |IAP| services in the
database. From here, click an |IAP| service to open details about it; additional credits can be
purchased from here.

On the following page, click the :guilabel:`Buy Credit` button. Doing so loads a :guilabel:`Buy
Credits for (IAP Account)` page in a new tab. From here, click :guilabel:`Buy` on the desired pack
of credits. Then, follow the prompts to enter payment details, and confirm the order.

   .. image:: in_app_purchase/buy-pack.png
      :align: center
      :alt: The SMS service page on IAP.Odoo.com with four packs of credits available for purchase.

Once the transaction is complete, the credits are available for use in the database.

.. _in_app_purchase/low-credits:

Low-credit notification
~~~~~~~~~~~~~~~~~~~~~~~

It is possible to be notified when credits are low, in order to avoid running out of credits, while
using an |IAP| service. To do that, follow this process:

Go to the :menuselection:`Settings application` and type `IAP` in the search bar. Under the Contacts

> section, where it says :guilabel:`Odoo IAP`, click :guilabel:`View My Services`.

Reviewer: i think Contacts can have a GUI because it is ON the UI. If not, I would italicize it.

The available |IAP| accounts appear in a list view on the :guilabel:`IAP Account` page. From here,
click on the desired |IAP| account to view that service's details.

On the details page, tick the :guilabel:`Receive threshold warning` box. Doing so reveals two
fields on the form: :guilabel:`Warning Threshold` and :guilabel:`Contact Email`.

In the :guilabel:`Warning Threshold` field, enter an amount of credits Odoo should use as the minimum

> threshold for this service. In the :guilabel:`Contact Email` field, enter the email address that
> should receive the notification.

Reviewer: ```suggestion
threshold for this service. In the :guilabel:`Contact Email` field, enter the email address that
receives the notification.
```

=====================

> In-app purchase (IAP)

Reviewer 2: ```suggestion
In-app purchases (IAP)
```

the service in the app to activate it. However, each service requires its own prepaid credits, and
when they run out, users **must** :ref:`buy more <iap/buying_credits>` in order to keep using it.

.. note::
   Enterprise Odoo users with a valid subscription get free credits to test IAP features before
   deciding to purchase more credits for the database. This includes demo/training databases,
   educational databases, and one-app-free databases.

.. _in_app_purchase/portal:

IAP services
============

|IAP| services are provided by Odoo, as well as third-parties, and have a wide range of uses.

The following |IAP| services are offered by Odoo:

- :guilabel:`Documents Digitization`: digitizes scanned or PDF vendor bills, expenses, and resumes
  with OCR and AI.
- :guilabel:`Partner Autocomplete`: automatically populates contact records with corporate data.
- :guilabel:`SMS`: sends SMS text messages to contacts directly from the database.
- :guilabel:`Lead Generation`: generates leads based on a set of criteria, and converts web visitors
  into quality leads and opportunities.
- :guilabel:`Snailmail`: sends customer invoices and follow-up reports by post, worldwide.

> - :guilabel:`Signer identification with itsme®️`: ask document signatories in Odoo Sign to provide

Reviewer 2: ```suggestion
- :guilabel:`Signer identification with itsme®️`: ask document signatories in Odoo *Sign* to provide
```

>   their identity using the itsme® identity platform. Available in Belgium and in the Netherlands.

Reviewer 2: ```suggestion
  their identity using the *itsme®* identity platform, which is available in Belgium and the Netherlands.
```

> For more information on every service currently available (offered from other developers other than

Reviewer 2: ```suggestion
For more information on every service currently available (offered from developers other than
```

Odoo), visit the `Odoo IAP Catalog <https://iap.odoo.com/iap/all-in-app-services>`_.

Use IAP services
----------------

|IAP| services are automatically integrated with Odoo, and do **not** require users to configure any
settings. To use a service, simply interact with it wherever it appears in the database.

.. example::
   The following flow focuses on the *SMS* |IAP| service being used from a contact's record.

   This can be done by clicking the :guilabel:`📱 (phone) SMS` icon within the database.

   .. image:: in_app_purchase/sms-icon.png
      :align: center
      :alt: The SMS icon on a typical contact information form located within an Odoo database.

   One way to utilize the *SMS* |IAP| service with Odoo is showcased in the following steps:

   First, navigate to the :menuselection:`Contacts application`, and click on a contact with a
   mobile phone number entered in either the :guilabel:`Phone` or :guilabel:`Mobile` field of the
   contact form.

   Next, find the :guilabel:`📱 (phone) SMS` icon that appears to the right of the :guilabel:`Phone`
   or :guilabel:`Mobile` fields. Click the :guilabel:`📱 (phone) SMS` icon, and a :guilabel:`Send

>    SMS Text Message` pop-up window appears.

Reviewer 2: I'm not seeing the pop-up window titled anything but "Odoo" in my 15.0 runbot that I'm using for this review. Would you mind double-checking and confirming this title (if there is one) during your revisions? Thanks!

>    Finally, type a message in the :guilabel:`Message` field of the pop-up window. Then, click the

Reviewer 2: ```suggestion
   Type a message in the :guilabel:`Message` field of the pop-up window. Then, click the
```

Credits are purchased in *Packs* from the `Odoo IAP Catalog
<https://iap.odoo.com/iap/all-in-app-services>`_, and pricing is specific to each service.

.. example::
   The `SMS service <https://iap.odoo.com/iap/in-app-services/1>`_ has four packs available, in
   denominations of:

   - :guilabel:`Starter Pack`: 10 credits
   - :guilabel:`Standard Pack`: 100 credits
   - :guilabel:`Advanced Pack`: 500 credits
   - :guilabel:`Expert Pack`: 1,000 credits

   .. image:: in_app_purchase/packs.png
      :align: center
      :alt: Four different packs of credits for the SMS IAP service.

   The number of credits consumed depends on the length of the SMS and the country of destination.

   For more information, refer to the :doc:`SMS Pricing and FAQ
   </applications/marketing/sms_marketing/pricing/pricing_and_faq>` documentation.

.. _iap/buying_credits:

Buy credits
-----------

If there are not enough credits to perform a task, the database automatically prompts the purchase
of more credits.

Users can check the current balance of credits for each service, and manually purchase more credits,

> by navigating to the :menuselection:`Settings app --> Contacts --> Odoo IAP --> View My Services`.

Reviewer 2: ```suggestion
by navigating to the :menuselection:`Settings app --> Contacts section`, and beneath the :guilabel:`Odoo IAP` setting, click :guilabel:`View My Services`.
```

> Doing so reveals an :guilabel:`My Services` page, listing the various |IAP| services in the

Reviewer 2: ```suggestion
Doing so reveals a :guilabel:`My Services` page, listing the various |IAP| services in the
```

database. From here, click an |IAP| service to open its :guilabel:`Account Information` page, where
additional credits can be purchased.

Manually buy credits
~~~~~~~~~~~~~~~~~~~~

To manually buy credits in Odoo, follow these steps:

> First, go to the :menuselection:`Settings application` and type `IAP` in the search bar.

Reviewer 2: ```suggestion
First, go to the :menuselection:`Settings application`, and type `IAP` in the :guilabel:`Search...` bar.
```

> Alternatively scroll down to the :guilabel:`Contacts` section. Under the :guilabel:`Contacts`

Reviewer 2: ```suggestion
Alternatively, users can scroll down to the :guilabel:`Contacts` section. In the :guilabel:`Contacts`
```

   .. image:: in_app_purchase/view-services.png
      :align: center
      :alt: The Settings app showing the Odoo IAP heading and View My Services button.

Doing so reveals an :guilabel:`IAP Account` page, listing the various |IAP| services in the
database. From here, click an |IAP| service to open details about it; additional credits can be
purchased from here.

On the following page, click the :guilabel:`Buy Credit` button. Doing so loads a :guilabel:`Buy
Credits for (IAP Account)` page in a new tab. From here, click :guilabel:`Buy` on the desired pack
of credits. Then, follow the prompts to enter payment details, and confirm the order.

   .. image:: in_app_purchase/buy-pack.png
      :align: center
      :alt: The SMS service page on IAP.Odoo.com with four packs of credits available for purchase.

Once the transaction is complete, the credits are available for use in the database.

.. _in_app_purchase/low-credits:

Low-credit notification
~~~~~~~~~~~~~~~~~~~~~~~

It is possible to be notified when credits are low, in order to avoid running out of credits, while
using an |IAP| service. To do that, follow this process:

> Go to the :menuselection:`Settings application` and type `IAP` in the search bar. Under the Contacts

Reviewer 2: ```suggestion
Go to the :menuselection:`Settings application`, and type `IAP` in the :guilabel:`Search...` bar. Under the :guilabel:`Contacts`
```

section, where it says :guilabel:`Odoo IAP`, click :guilabel:`View My Services`.

The available |IAP| accounts appear in a list view on the :guilabel:`IAP Account` page. From here,
click on the desired |IAP| account to view that service's details.

> On the details page, tick the :guilabel:`Receive threshold warning` box. Doing so reveals two

Reviewer 2: ```suggestion
On the details page, tick the :guilabel:`Receive threshold warning` checkbox. Doing so reveals two
```
